### DISCUSSION TITLE
Ogeysiis 📢

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#80] [Announcements]
Sida qorshaha ah bandhiga dhaqsoPay waa 19, 12, 2024 08:00PM, Waxa uuna noqon doonaa live.

Aan u diyaar garowno sido kalena aan dhameystirno inta ka hartay Insha'Allah! 

Mr Sharafdin

**Comment by shiinedev** [comment]
Waqti yar ba noo haray in kasto wax review ah anan ka heynin  si hdi ay wax qaldan jiran u saxno bro

**Reply by sharafdin** [reply]
galabta kuso qabaa bro

**Reply by shiinedev** [reply]
Ok bro

**Comment by hayle01** [comment]
Insha allah wax aan sidaas u buurnayn ayaa inoo haray.
